pkg/operator/sync_common.go

```diff
@@ -177,7 +178,7 @@ func manageConfigMapCABundleControllerDeployment(client appsclientv1.AppsV1Inter
 func manageDeployment(client appsclientv1.AppsV1Interface, eventRecorder events.Recorder, options *operatorv1.ServiceCA, resourcePath string, forceDeployment bool) (bool, error) {
 	required := resourceread.ReadDeploymentV1OrDie(v4_00_assets.MustAsset(resourcePath + "deployment.yaml"))
 	required.Spec.Template.Spec.Containers[0].Image = os.Getenv("CONTROLLER_IMAGE")
-	required.Spec.Template.Spec.Containers[0].Args = append(required.Spec.Template.Spec.Containers[0].Args, fmt.Sprintf("-v=%s", options.Spec.LogLevel))
+	required.Spec.Template.Spec.Containers[0].Args = append(required.Spec.Template.Spec.Containers[0].Args, fmt.Sprintf("-v=%d", loglevel.LogLevelToKlog(options.Spec.LogLevel)))
```

Comment:
This is not only about breakage with empty value, but about the API semantics. Before we wanted numbers as string type, now Normal, Debug, Trace, TraceAll.

Was this just broken before?

Reply:
Looking at the API, it was broken.

Reply:
Is this comment actionable?

Reply:
No. You are fixing the feature.